global.json

```diff
@@ -1,5 +1,7 @@
 {
   "sdk": {
-    "version": "5.0.100-preview"
+    "version": "5.0.100-preview",
+    "allowPrerelease": true,
+    "rollForward": "major"
```

Comment:
Our plan would be to eventually remove this once the 5.0 SDK ships. 
It shouldn't matter too much which SDK is being used on the devs local machine.

Reply:
yes, remove global.json :D